strutil.h

```diff
@@ -92,6 +92,32 @@ namespace strutil
         return result;
     }
 
+    /**
+     * @brief Converts std::string to capitalize case.
+     * @param str - std::string that needs to be converted.
+     * @return Capitalized case input std::string.
+     */
+    static inline std::string to_capitalize(const std::string & str)
+    {
+        auto result = str;
+        result[0] = std::toupper(result[0]);
+
+        return result;
+    }
+
+    /**
+     * @brief Converts std::string to capitalize case and with only the firsr letter as capitalize.
+     * @param str - std::string that needs to be converted.
+     * @return Capitalized case input std::string.

```

Comment:
Change to `@return A string with the first letter capitalized. All other characters stay unchanged. It doesn't modify the input string.`.